website/docs/docs/build/udfs.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "User-defined functions"
+description: "Learn how to add user-defined functions (UDFs) to your dbt projects."
+id: "udfs"
+---
+
+# User-defined functions <Lifecycle status="beta" />
+
+:::info
+UDFs are not yet supported in the dbt Fusion engine.
+:::
+
+User-defined functions (UDFs) enable users to define and register custom functions in your warehouse. Like [macros](/docs/build/jinja-macros), UDFs promote code reuse. They run natively in the warehouse so you can reuse the same logic in tools outside dbt.
+
+dbt creates, updates, and renames UDFs as part of DAG execution. The UDF file is created before building the model that references it.
+
+## Supported adapters
```

Comment:
@luna-bianca @QMalcolm Are UDFs backwards compatible to previous Core versions? If not, we might want to add as a prereq.